torch/_export/converter.py

```diff
@@ -571,17 +673,30 @@ def __init__(
         self.param_names: Set[str] = {name for name, _ in ts_model.named_parameters()}
         self.buffer_names: Set[str] = {name for name, _ in ts_model.named_buffers()}
 
+        # Populate nn module parameters and buffers.
+        self.mod_param_and_buffer_map: Dict[str, Any] = dict()
+        for name, param in ts_model.named_parameters():
+            self.mod_param_and_buffer_map[name] = param
+        for name, buffer in ts_model.named_buffers():
+            self.mod_param_and_buffer_map[name] = buffer
+
     def convert(self) -> ExportedProgram:
+        blocks_to_lifted_attrs = get_block_lifted_args(self.ts_graph)
+
         graph_converter = TS2FXGraphConverter(
-            self.ts_graph, self.param_names, self.buffer_names
+            self.ts_graph,
+            self.param_names,
+            self.buffer_names,
+            self.mod_param_and_buffer_map,
+            blocks_to_lifted_attrs,
         )
         gm = graph_converter.convert()
         ep = self.retrace_as_exported_program(gm, graph_converter.tensor_constants)
         return ep
 
     def retrace_as_exported_program(self, gm: torch.fx.GraphModule, tensor_constants):
         # TODO: adjust input orders to match GraphSignature convention
-        inputs = [*self.sample_args, *self.params, *tensor_constants.values()]
+        inputs = [*self.sample_args]
```

Comment:
We can just delete this line now? And directly pass in `self.sample_args` to torch.export?